package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "nodejs-mentoring",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "task1": "nodemon ./task1.1/index.js",
+    "task2": "nodemon ./task1.2/index.js"
+  },
+  "author": "Mikita_Ustsiushenka@epam.com",
+  "devDependencies": {
+    "@babel/cli": "^7.13.14",
+    "@babel/core": "^7.13.15",
+    "@babel/node": "^7.13.13",
+    "@babel/preset-env": "^7.13.15",
```

Comment:
Currently babel is only installed as a dependence, but not used for code transpilation
Current task is actually a little bit outdated, as Node.js 14+ supports es6 modules by default (if type=module added to package.json), but babel can be also useful for other JS features which are not supported by Node.js for now

Reply:
so the task is to configure babel as a transpiler for you code

Reply:
@alligator2005 Thanks, updated